src/App.js

```diff
@@ -1,23 +1,43 @@
-import logo from './logo.svg';
 import './App.css';
+import Counter from "./components/counter/Counter";
+import { SearchForm } from "./components/search-form/SearchForm";
+import "@fontsource/montserrat";
+import "@fontsource/montserrat/400.css";
+import "@fontsource/montserrat/400-italic.css";
+import { GenreSelect } from "./components/genre-select/GenreSelect";
+import React from "react";
+
+export const listGenres = [
+    {
+    id: '1', name: 'Crime'
+    }, {
+    id: '2', name: 'Documentary'
+    }, {
+    id: '3', name: 'Horror'
+    }, {
+    id: '4', name: 'Comedy'
+    }
+]
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+        <h2 style={{textAlign: 'left'}}>Epam React Course</h2>
+        <hr/>
+        <div>
+            <header style={{textAlign: 'center'}}>Counter component</header>
+            <Counter/>
+        </div><br/><br/><hr/>
+        <div>
+            <header style={{textAlign: 'center'}}>SearchForm component</header><br/><br/>
+            <SearchForm
```

Comment:
The properties for `<SearchForm>` are missing: 
1. Initial search query. Use the value to set the initial value for the input
2. A "onSearch" callback property. Call the callback property every time the user presses Enter when the input has focus or when the user clicks the Search button. Pass current input value in callback arguments.